tests/tablespaces/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: "3.9"  # optional since v1.27.0
+services:
+  monitor:
+    build: .
+    environment:
+      PGDATA: /tmp/pgdata
+      PG_AUTOCTL_DEBUG: 1
+    command: pg_autoctl create monitor --ssl-self-signed --auth trust --run
+    expose:
+      - 5432
+    container_name: monitor
+  node1:
+    build: .
+    environment:
+      PGDATA: /var/lib/postgres/data
+      PG_AUTOCTL_DEBUG: 1
+    command: [
+    "pg_autoctl", "create", "postgres",
+    "--pgport", "5433",
+    "--ssl-self-signed",
+    "--auth", "trust",
+    "--pg-hba-lan",
+    "--hostname", "node1.tablespaces_default",
+    "--monitor", "postgresql://autoctl_node@monitor/pg_auto_failover",
+    "--run"]
```

Comment:
Is it possible to use a very short command and move all the options to the environment, as done in https://github.com/citusdata/pg_auto_failover/blob/93f995812dffef29695f6f0ce63c5e69f5a34a74/tests/upgrade/docker-compose.yml#L24 for instance?

Reply:
On it

Reply:
I found a small behavior difference when working on this between env variable and function args when creating/running a data node:

`--pgport=1234` if the monitor is passed in with `PG_AUTOCTL_MONITOR` or `--monitor` without a port, it assumes the default port for the monitor of 5432

`PGPORT=1234` if the monitor is passed in with `PG_AUTOCTL_MONITOR` or `--monitor` without a port, it constructs a monitor URI with _no_ port (e.g. `postgresql://autoctl_node@monitor:/pg_auto_failover` and connections fail. passing in the monitor port in the monitor string works, but this is surprising.

Is this desired for some postgres-y reason I am not familiar with, or shall I file an issue?